guides/doc-Upgrading_Project/master.adoc

```diff
@@ -17,6 +17,13 @@ ifdef::satellite[]
 include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
 endif::[]
 
+ifdef::foreman-el[]
+[NOTE]
+====
+If you use the Katello plugin, please use {BaseURL}Upgrading_Project/index-katello.html[Upgrading Foreman with Katello].
```

Comment:
```suggestion
If you have installed the Katello plugin, you must use {BaseURL}Upgrading_Project/index-katello.html[Upgrading Foreman with Katello].
```
I think this should be a bit more forceful.

Reply:
Applied.